### DISCUSSION TITLE
[Multilingual Associations] without [Tags]?

### DISCUSSION THREAD

**Discussion by 4digitalworld** [joomla/joomla-cms#28012] [2. Cold Features]
### Is your feature request related to a problem? Please describe.

There are over 600 `tags` and over 300 `categories` on my website now.

I just found that there are no **button** for creating `Associations` in the `Tags` component.

I started translating my website into another language and now I started creating `Tags`, and I understand that a more convenient way to create tags in another language is to create using the `Associations` **button**. I did this for `Categories`, `Menus`, and `Articles`, but cannot do this for `Tags`.

I think it would be logical if such a button were also for `Tags`.

![изображение](https://user-images.githubusercontent.com/3662519/75091332-1fbd6400-557d-11ea-8263-716518ca873e.png)

### Describe the solution you'd like
![изображение](https://user-images.githubusercontent.com/3662519/75091291-bb9aa000-557c-11ea-8445-f338d2b6c9c2.png)

I suggest adding this `Associations` button when creating `Tags` in another language.

### Additional context

I add different **Tags** (**for example:** `ru` - `Компьютер`, `en` - `Computer`), which are written in different languages for `Articles` on the website. I do the same with `Categories`, but when I created `Categories` in another language, it was very convenient using the `Association` button. Now I can’t also conveniently and quickly create `Tags` in another language. This is a kind of problem when you need to create 600 `Tags` in another language.

**Comment by infograf768** [comment]
The tags component is not coded to use multilingual associations.
Adding the Associations button will not magically transforming it into a multilingual associations aware extension.
You are welcome to give it a try and propose a patch, but that would be for J4 as we do not add new functionalities in J3.
